daemon-scheduler/pkg/engine/scheduler.go

```diff
@@ -37,20 +39,20 @@ const (
 )
 
 type scheduler struct {
-	id             string
-	ctx            context.Context
-	environmentSvc deployment.Environment
-	deploymentSvc  deployment.Deployment
-	css            facade.ClusterState
-	ecs            facade.ECS
-	events         chan<- Event
-	executionState map[string]environmentExecutionState
-	inProgress     bool
-	inProgressLock sync.RWMutex
+	id                 string
+	ctx                context.Context
+	environmentService environment.EnvironmentService
```

Comment:
Yup, I'll rename `deploymentSvc` to `deploymentService` when I do the deployments refactor.